copy/multiple.go

```diff
@@ -119,7 +138,10 @@ func (c *copier) copyMultipleImages(ctx context.Context, policyContext *signatur
 	// Copy each image, or just the ones we want to copy, in turn.
 	instanceDigests := updatedList.Instances()
 	instanceEdits := []internalManifest.ListEdit{}
-	instanceCopyList := prepareInstanceCopies(instanceDigests, options)
+	instanceCopyList, err := prepareInstanceCopies(instanceDigests, options)
+	if err != nil {
+		return nil, fmt.Errorf("while preparing instances for copy: %w", err)
+	}
 	c.Printf("Copying %d of %d images in list\n", len(instanceCopyList), len(instanceDigests))
```

Comment:
This might now say “copying 5 of 3 images”, so the wording could be improved, here and probably in the other  messages around, so that we don’t refer to “image 5/3 in manifest list” and the like.